src/Renderer/ThreeExtended/FeaturesUtils.js

```diff
@@ -0,0 +1,122 @@
+function pointIsOverLine(point, linePoints, precision) {
+    const x0 = point._values[0];
+    const y0 = point._values[1];
+    // in first j is last point of line and then it'll be (i - 1)
```

Comment:
Ok. 
How about adding: `// for each segment of the line (j is i -1)` 

Also, your code assumes that this is a closed line; I'm not sure this is a valid assumption.

Reply:
I change that, to no closed line